README.md

```diff
@@ -22,8 +22,42 @@ Imports and initializes Smartlook recorder into a page.
 
     ```
     init(string key)
-    init(string key, params?: { region?: 'eu' | 'us', version?: 'legacy' | 'nextgen', cookies?: boolean })
-    ```
+    init(string key, params?: { region?: 'eu' | 'us', cookies?: boolean, relayProxyUrl?: string })
+    ```
+	Required parameters:
+	* key:
+		* Obtained in application at project settings
+
+	Optional parameters:
+
+	* region:
+		* supported values:
+			* `'eu'`
+			* `'us'`
+		* example:
+			* `smartlookClient.init('xxxxx', { region: 'us' })`
+		* description
+			* Region where data will be captured and stored
+			* **Do not change** unless told by your sales manager
+	* cookies:
+		* supported values:
+			* `true`
+			* `false`
+		* example:
+			* `smartlookClient.init('xxxxx', { cookies: false })`
+		* description
+			* Use false if you do not want to store recording metadata in cookies
+			* Note that disabling cookies with block the ability to connect visitors between domain and its subdomains.
```

Comment:
maybe add also link https://help.smartlook.com/en/articles/6064963-cookies-in-smartlook for more info